content/en/docs/acknowledgements/_index.md

```diff
@@ -8,7 +8,7 @@ weight: 200
 We would like to acknowledge the following sources for some of the content on this site:
 
 1. [APM is Dying and That's Okay - Lightstep](https://lightstep.com/blog/apm-is-dying-and-thats-okay)
-2. [Tracing Concepts - github/cartermp](https://gist.github.com/cartermp/d4202c531bf73e369fb945e53a1a0045)
+2. [Alexandria Pigram](https://github.com/alexandriastech) via [Honeycomb](https://www.honeycomb.io/) for tracing content [OTel Concepts](/docs/concepts/otel-concepts)
```

Comment:
Btw, there are additional diagrams on the OTel Concepts page, which were not a part of Alexandria's contributions.

```suggestion
2. [Alexandria Pigram](https://github.com/alexandriastech) via [Honeycomb](https://www.honeycomb.io/) for tracing content in [OTel Concepts](/docs/concepts/otel-concepts)
```